recipes/uni-algo/all/conanfile.py

```diff
@@ -75,7 +75,7 @@ def validate(self):
                 raise ConanInvalidConfiguration(
                     f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
                 )
-        if is_msvc(self):
+        else:
             check_min_vs(self, int(self._compilers_minimum_version["msvc"]))
```

Comment:
this logic is broken for `compiler=Visual Studio` (it will always raise), you can remove this branching and the specific `check_min_vs()`, and only keep a check for is_msvc & shared.

Reply:
Well... https://github.com/conan-io/conan-center-index/pull/17093